### PR TITLE
Use stdlib tomllib for Python 3.11+

### PR DESCRIPTION
**Pull request by hugovk**
And test on Python `3.11-dev`, currently the beta.

### CODE REVIEW

**Review by mgedmin — APPROVED** [review]
LGTM, except for the `isort` failure in CI.

isort is not the smartest cookie on the block and doesn't understand imports inside try/except blocks; the easiest solution is to move them below all other import blocks.

No, actually the easiest solution is to run `tox -e isort -- *.py` and commit the changes it makes.

**Comment by mgedmin** [comment]
Hm, mypy is also unhappy.

File: check_manifest.py [hunk 1]
```diff
@@ -34,7 +34,12 @@
 from typing import List, Optional, Union
 from xml.etree import ElementTree as ET
 
-import tomli
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
```
Comment (mgedmin) [inline check_manifest.py:41]: Whee, now we have test coverage enforcement issues!

Both branches need a `# pragma: nocover`

Comment (hugovk) [reply check_manifest.py]: Updated!

**Pull request merged**